newIDE/app/src/Utils/PIXITicker.js

```diff
@@ -3,6 +3,9 @@ import * as PIXI from 'pixi.js-legacy';
 // PIXI has a ticker that is used by PIXI InteractionManager, and which
 // frequently check if interaction happened. We may want to disable it
 // when it's useless to do these interaction checks to save CPU usage.
+// TODO: We might not need this anymore since PIXI v7 dropped the use of
+// InteractionManager in favor of EventSystem that does not seem
+// to use a ticker.
```

Comment:
I read about this too and unsure we need this indeed.

Reply:
Let's drop this later when we'll be sure it's not needed. Out of caution, we'll keep it for now, it's not difficult to clean this later.